www/install/sql/centreon/Update-DB-18.10.4_to_19.04.1.sql

```diff
@@ -0,0 +1,20 @@
+-- Add new configuration for Centeron Broker : Use multiple thread to connect to database
+INSERT INTO `cb_field` (`cb_field_id`, `fieldname`, `displayname`, `description`, `fieldtype`, `external`) VALUES
+(75, 'connections_count', 'Connections count', 'Number of opened connection to the database.', 'int', NULL),
+(76, 'enable_command_cache', 'Enable command cache', 'Cache the executed command for better insert performance. Warning: This options use more memory.', 'radio', NULL);
```

Comment:
The cache filter is no more configurable in centreon-broker. This configuration part must be removed.